File: server/computed_functions/03_startup_tokens.compute.mjs
// 03_startup_tokens — STARTUP. Composite returning { case_001, case_005, timestamp_iso }.
// All three share one timestamp because the function evaluates once at boot.
const ts = Date.now();
const tsIso = new Date(ts).toISOString();

export default function compute(_ctx, _path) {
  return {
    case_001: `startup-001-${ts}`,
    case_005: `startup-005-${ts}`,
    timestamp_iso: tsIso,
  };
}

export const scope = "STARTUP";

File: server/computed_functions/03_startup_tokens.compute.py
"""03_startup_tokens — STARTUP. {case_001, case_005, timestamp_iso}, single shared timestamp."""
from __future__ import annotations
import time
from datetime import datetime, timezone

_ts = int(time.time() * 1000)
_ts_iso = datetime.fromtimestamp(_ts / 1000, tz=timezone.utc).isoformat()


def compute(_ctx, _path=None):
    return {
        "case_001": f"startup-001-{_ts}",
        "case_005": f"startup-005-{_ts}",
        "timestamp_iso": _ts_iso,
    }


scope = "STARTUP"
